workspace_host/interface.js

```diff
@@ -323,9 +323,10 @@ async.series([
  */
 async function pushContainerContentsToS3(workspace) {
     const workspacePath = path.join(workspacePrefix, `workspace-${workspace.launch_uuid}`);
+    const s3Path = path.join(config.workspaceS3Bucket, `workspace-${workspace.id}-${workspace.version}`);
```

Comment:
I think this should have `current/` on the end, right?

Reply:
🤦‍♂️ I think so

Reply:
See #2975